### PR TITLE
Document the module collection.

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: src/collection.rs [hunk 1]
```diff
@@ -14,27 +14,27 @@
...
 use std::marker::PhantomData;
 use std::ops;
-use std::path;
 use std::result::Result as StdResult;
 use std::slice;
 
-use Result;
-use objects::AddPrefix;
-
+/// An object that have a unique identifier.
```
Comment (datanel) [inline src/collection.rs]: that has

File: src/gtfs/mod.rs [hunk 2]
```diff
@@ -44,11 +46,20 @@ fn add_prefix_to_collections(prefix: String, collections: &mut Collections) -> R
...
 
-pub fn read<P: AsRef<path::Path>>(
-    path: P,
-    config_path: Option<P>,
-    prefix: Option<String>,
-) -> Result<Model> {
+/// Imports a `Model` from the [GTFS](http://gtfs.org/) files in the
+/// `path` directory.
+///
+/// The `config_path` argument allow you to give a file containing a
```
Comment (datanel) [inline src/gtfs/mod.rs]: allows

File: get-corresponding-derive/src/lib.rs [hunk 3]
```diff
@@ -65,16 +65,21 @@ fn impl_get_corresponding(ast: &syn::DeriveInput) -> quote::Tokens {
             }
         });
         quote! {
+            /// A trait to get the objects corresponding to a set of objects.
```
Comment (nonifier) [inline get-corresponding-derive/src/lib.rs]: Adjectives before nouns : `to get the corresponding objects`.

File: get-corresponding-derive/src/lib.rs [hunk 4]
```diff
@@ -65,16 +65,21 @@ fn impl_get_corresponding(ast: &syn::DeriveInput) -> quote::Tokens {
...
             pub trait GetCorresponding<T: Sized> {
+                /// for the given self, returns the set of
+                /// corresponding `T` indices.
                 fn get_corresponding(&self, &#name) -> IdxSet<T>;
             }
             impl #name {
+                /// Gets the set of `U` indices corresponding to the `from` set.
```
Comment (nonifier) [inline get-corresponding-derive/src/lib.rs]: Same here: `corresponding` << >> `indices`

File: src/gtfs/mod.rs [hunk 5]
```diff
@@ -44,11 +46,20 @@ fn add_prefix_to_collections(prefix: String, collections: &mut Collections) -> R
...
 
-pub fn read<P: AsRef<path::Path>>(
-    path: P,
-    config_path: Option<P>,
-    prefix: Option<String>,
-) -> Result<Model> {
+/// Imports a `Model` from the [GTFS](http://gtfs.org/) files in the
+/// `path` directory.
+///
+/// The `config_path` argument allows you to give a file containing a
```
Comment (nonifier) [inline src/gtfs/mod.rs]: It's not very clear whether `config_path` is a path to a directory that contains a Json, or a path to the Json file itself.

**Pull request merged**